guides/common/attributes-base.adoc

```diff
@@ -76,6 +76,7 @@
 :package-remove: dnf remove
 :package-update-project: dnf update
 :package-update: dnf update
+:PAT: Personal Access Token
```

Comment:
I haven't found a case where you overwrite PAT > Do you really need this as an attribute?